spec/cocina/models/mapping/to_mods/note_spec.rb

```diff
@@ -42,4 +43,23 @@
       XML
     end
   end
+
+  context 'when notes has a parallelValue' do
+    let(:notes) do
+      [
+        Cocina::Models::DescriptiveValue.new(parallelValue: [Cocina::Models::DescriptiveValue.new(value: 'monthly', type: 'frequency')])
```

Comment:
Shouldn't a parallelValue have at least 2 values?

Reply:
Maybe, in the case I'm running into there is only one.